config/c-code/tests.yml.j2

```diff
@@ -130,6 +128,9 @@ jobs:
           && startsWith(github.ref, 'refs/tags')
           && startsWith(runner.os, 'Mac')
           && !startsWith(matrix.python-version, 'pypy')
+{% if with_future_python %}
+          && ${{ matrix.python-version != '%(future_python_version)s' }}
```

Comment:
Why are `${{` ... `}}` needed here, but not in other parts of the same conditional expressions that refer to `matrix.python-version`?

Reply:
I fixed it in a later commit as it did not work as expected, it made the whole expression to become `true`.